sqlparse/filters.py

```diff
@@ -561,8 +563,231 @@ def process(self, stack, stream):
                     mode = 1
 
 
+ColumnAttributeProperty = namedtuple('ColumnAttributeProperty', 'has_value is_type_attribute')
+
+
+class MysqlCreateStatementFilter(object):
```

Comment:
It seems that this class is doing some of grouping jobs. It probably makes more sense to define grouping functions for column definition, column data type, attributes, table create definition, etc., These grouping functions then can be used to create corresponding statement token list. There seems no need to create a custom token such as column name, etc.

It could have a custom statement token list derived from existing Statement token list if you'd like. Things such as table name, columns could be the functions/properties of that custom statement.